ament_cmake_core/cmake/symlink_install/ament_cmake_symlink_install.cmake.in

```diff
@@ -268,16 +268,31 @@ function(_ament_cmake_symlink_create_symlink absolute_file symlink)
     file(REMOVE "${symlink}")
   endif()
 
-  execute_process(
-    COMMAND "@CMAKE_COMMAND@" "-E" "create_symlink"
-      "${absolute_file}"
-      "${symlink}"
-  )
-  # the CMake command does not provide a return code so check manually
-  if(NOT EXISTS "${symlink}" OR NOT IS_SYMLINK "${symlink}")
-    get_filename_component(destination "${symlink}" REALPATH)
-    message(FATAL_ERROR
-      "Could not create symlink '${symlink}' pointing to '${absolute_file}'")
+  if(NOT WIN32)
+    execute_process(
+      COMMAND "@CMAKE_COMMAND@" "-E" "create_symlink"
+        "${absolute_file}"
+        "${symlink}"
+    )
+    # the CMake command does not provide a return code so check manually
+    if(NOT EXISTS "${symlink}" OR NOT IS_SYMLINK "${symlink}")
+      get_filename_component(destination "${symlink}" REALPATH)
+      message(FATAL_ERROR
+        "Could not create symlink '${symlink}' pointing to '${absolute_file}'")
+    endif()
+  else()
+    string(REPLACE "/" "\\" symlink "${symlink}")
+    string(REPLACE "/" "\\" absolute_file "${absolute_file}")
```

Comment:
That's right, but those lines may not end up being needed. This pull request isn't really ready for review since it isn't working yet.